lua/wire/zvm/zvm_opcodes.lua

```diff
@@ -265,7 +265,11 @@ ZVM.OpcodeTable[31] = function(self)  --CALL
   self.PrecompileBreak = true
 end
 ZVM.OpcodeTable[32] = function(self)  --BNOT
-  self:Dyn_EmitOperand("VM:BinaryNot($1)")
+  self:Dyn_Emit("if VM.IPREC > 32 then")
+    self:Dyn_EmitOperand("VM:BinaryNot($1)")
+  self:Dyn_Emit("else")
+    self:Dyn_EmitOperand("VM:ClampBinaryToIPREC(bit.bnot($1,$2))")
```

Comment:
bit.bnot only takes one arg right?

Reply:
Yeah, will get a commit out to fix it after I'm sure whether to use ldexp or pow